docs/api/Chip/Chip.md

```diff
@@ -15,9 +15,8 @@ Props
 | avatar | node |  | Avatar element. |
 | className | string |  | CSS `className` of the root element. |
 | deleteIconClassName | string |  | CSS `className` of the delete icon element. |
-| label | string |  | Label |
+| label | string |  | The label text. |
 | labelClassName | string |  | CSS `className` of the label. |
-| onClick | function |  | Callback function fired when the `Chip` element is clicked. If set, the chip will by styled for hover focus and active states.<br><br>**Signature:**<br>`function(event: object) => void`<br>*event:* `onClick` event targeting the root element. |
```

Comment:
Thanks - I was in two minds about that one, as we had previously decided not to document standard DOM events, but was trying to find somewhere to document that using this triggers changes to the button style. I guess it's implicit in the docs demo.